target_postgres/singer_stream.py

```diff
@@ -19,6 +19,12 @@
 SINGER_LEVEL = '_sdc_level_{}_id'
 SINGER_VALUE = '_sdc_value'
 
+DATAMILL_RAW_LINE_SIZE = '_dm_raw_line_size'
```

Comment:
This almost makes it seem like it's persisted, which it isn't? Maybe just `__raw_line_size`? As long as it's not persisted, I wouldn't worry about namespacing.

Reply:
Good point. `__` should be fine for ensuring the payload itself occludes any conflicting keys.